ReactAndroid/src/main/java/com/facebook/react/uimanager/layoutanimation/PositionAndSizeAnimation.java

```diff
@@ -21,8 +21,8 @@
   public PositionAndSizeAnimation(View view, int x, int y, int width, int height) {
     mView = view;
 
-    mStartX = view.getX();
```

Comment:
Consider adding a comment, when reading this code it's not obvious to me why the translation has to be subtracted.